src/api/app/models/channel.rb

```diff
@@ -163,9 +163,9 @@ def add_channel_repos_to_project(tpkg, mode = nil)
 
     # defined in channel
     self.channel_targets.each do |ct|
-      next if mode == :skip_disabled and ct.disabled
+      next if mode == :skip_disabled && ct.disabled
       repo_name = ct.repository.extended_name
-      next unless mode==:enable_all or not ct.disabled
+      next unless mode==:enable_all || !ct.disabled
```

Comment:
I agree though the code was like that before. So let's not make it too hard to get things into master;-)